lighthouse-core/config/config-helpers.js

```diff
@@ -214,7 +214,75 @@ function resolveModule(moduleIdentifier, configDir, category) {
        ${relativePath}`);
 }
 
+/**
+ * Many objects in the config can be an object whose properties are not serializable.
+ * We use a shallow clone for these objects instead.
+ * Any value that isn't an object will not be cloned.
+ *
+ * @template T
+ * @param {T} item
+ * @return {T}
+ */
+function shallowClone(item) {
```

Comment:
this function used to be called `cloneArrayWithPluginSafety` (we used to call the custom audit/gatherer definition "plugin support") which became very misleading once we *actually* introduced plugin support that means something else. This is just more straightforward for what it was.